readme.md

```diff
@@ -139,7 +168,16 @@ Define a Javascript constructor called 'Instructor'. Every instance of Instructo
 Instantiate an instructor named 'Andy' and call its `receivePresent` method with "Resin Laying Deer Figurine, Gold" as the argument.
 
 ```js
-// Your code starts here...
+var Instructor = function(name){
+  this.name = name;
+}
+
+Instructor.prototype = function receivePresent(gift){
+  console.log(this.name + "promptly drops the " + gift + "on the floor.");
+}
+
+var Andy = Instructor(Andy)
+Andy.receivePresent(Gold deer)
 
 ```
 
```

Comment:
When defining prototypal methods we write them out like so...

```js
Instructor.prototype.receivePresent = function(gift){
  console.log(this.name + " promptly drops the " + gift + " on the floor.");
}
```

So we're defining a method that is attached to the `Instructor` prototype object. We are not defining the `Instructor` prototype altogether.

(And don't forget to pass a string as an argument into the `Instructor` constructor!)